doc/main.html

```diff
@@ -148,50 +150,103 @@ <h4>Overall structure</h4>
           <caption>Ordered sequence of <code>section</code> elements comprising the <code>body</code> element.</caption>
           <thead>
             <tr>
-              <th>Cardinality</th>
-              <th>Section</th>
+              <th>Section Number</th>
+              <th>Section Name</th>
+              <th>Mandatory, Conditional, or Optional</th>
             </tr>
           </thead>
           <tbody>
             <tr>
-              <td>0..1</td>
-              <td><a>introduction section</a></td>
+              <td>Unnumbered</td>
+              <td><a>Table of contents</a></td>
+              <td>Mandatory (generated automatically)</td>
+            </tr>
+            <tr>
+              <td>Unnumbered</td>
+              <td><a>Foreward</a></td>
+              <td>Mandatory</td>
+            </tr>
+            <tr>
+              <td>Unnumbered</td>
+              <td><a>Introduction</a></td>
+              <td>Optional</td>
             </tr>
             <tr>
               <td>1</td>
-              <td><a>scope section</a></td>
+              <td><a>Scope</a></td>
+              <td>Mandatory</td>
             <tr>
-              <td>0..1</td>
-              <td><a>conformance section</a></td>
+              <td>2</td>
+              <td><a>Conformance</a></td>
+              <td>Mandatory</td>
             </tr>
             <tr>
-              <td>0..1</td>
-              <td><a>normative references section</a></td>
+              <td>3</td>
+              <td><a>Normative references</a></td>
+              <td>Mandatory</td>
             <tr>
-              <td>0..1</td>
-              <td><a>terms and definitions section</a></td>
+              <td>4</td>
+              <td><a>Terms and definitions</a></td>
+              <td>Mandatory</td>
             </tr>
             <tr>
-              <td>0..*</td>
-              <td><a>prose section</a></td>
+              <td>5-*</td>
+              <td>* (<a>Prose section</a>)</td>
+              <td>Mandatory</td>
             </tr>
             <tr>
-              <td>0..*</td>
-              <td><a>annex section</a></td>
+              <td>Annex A-*</td>
+              <td>* (<a>Annex section</a>)</td>
+              <td>Optional</td>
             </tr>
             <tr>
-              <td>0..1</td>
-              <td><a>bibliography section</a></td>
+              <td>Unnumbered</td>
+              <td><a>Bibliography</a></td>
+              <td>Conditional (if informative references present)</td>
             </tr>
           </tbody>
         </table>
 
+        <p class="note">For further section infomation and requirements outside of this document, see <a href="#bib-iso-directives-part2"></a> and <a href="#bib-smpte-ag-16"></a></p>
+
+      </section>
+
+      <section id="sec-toc-section">
```

Comment:
I am not a big fan of having this here this since it is auto-generated.

Reply:
Agreed, removed.